specifyweb/frontend/js_src/package.json

```diff
@@ -16,16 +16,16 @@
   },
   "scripts": {
     "//": "All of these should be executed from the js_src directory",
-    "localizationTests": "node --loader ts-node/esm lib/localization/__tests__/scanUsages.js",
-    "pullFromWeblate": "node --loader ts-node/esm lib/localization/utils/pull.js",
-    "schema:extract": "node --loader ts-node/esm lib/localization/schema-localization/extract.js",
-    "schema:pullFromWeblate": "node --loader ts-node/esm lib/localization/schema-localization/pull.js",
+    "localizationTests": "node --loader @swc-node/register/esm lib/localization/__tests__/scanUsages.ts",
```

Comment:
it's cool that we can completely migrate to a different typescript runner by just changing a package.json files - no other files need be changed!